engine/security/src/main/scala/pl/touk/nussknacker/ui/security/oauth2/OAuth2ClientApi.scala

```diff
@@ -43,7 +51,7 @@ class OAuth2ClientApi[ProfileResponse: Decoder, AccessTokenResponse: Decoder]
     val headers = configuration.headers ++ Map(configuration.authorizationHeader -> s"Bearer $accessToken")
 
     basicRequest
-      .contentType(contentType.toString)
+      .contentType(MediaType.ApplicationJson)
```

Comment:
this will always be json?

Reply:
in fact, this should be "accept", since this is a GET request, but I think I'll just remove it..

Reply:
hmm just got into this: https://docs.github.com/en/free-pro-team@latest/developers/apps/authorizing-oauth-apps#response

Reply:
ok, but I see accept is set in build.sbt: https://github.com/TouK/nussknacker/blob/staging/build.sbt#L248